src/ServiceManagerBaseStorage.sol

```diff
@@ -35,19 +37,33 @@ abstract contract ServiceManagerBaseStorage is IServiceManager, OwnableUpgradeab
     /// @notice The address of the entity that can initiate rewards
     address public rewardsInitiator;
 
-    /// @notice Sets the (immutable) `_avsDirectory`, `_rewardsCoordinator`, `_registryCoordinator`, and `_stakeRegistry` addresses
+    /// @notice The address of the slasher account
+    address public slasher;
+
+    /// @notice The address of the proposed slasher account
+    address public proposedSlasher;
+
+    /// @notice The timestamp when the slasher was proposed
+    uint256 public slasherProposalTimestamp;
+
+    /// @notice Boolean indicating if the migration has been finalized
+    bool public migrationFinalized;
+
+    /// @notice Sets the (immutable) `_avsDirectory`, `_rewardsCoordinator`, `_registryCoordinator`, `_stakeRegistry`, and `_allocationManager` addresses
     constructor(
         IAVSDirectory __avsDirectory,
         IRewardsCoordinator __rewardsCoordinator,
         IRegistryCoordinator __registryCoordinator,
-        IStakeRegistry __stakeRegistry
+        IStakeRegistry __stakeRegistry,
+        IAllocationManager __allocationManager
     ) {
         _avsDirectory = __avsDirectory;
         _rewardsCoordinator = __rewardsCoordinator;
         _registryCoordinator = __registryCoordinator;
         _stakeRegistry = __stakeRegistry;
+        _allocationManager = __allocationManager;
     }
 
     // storage gap for upgradeability
-    uint256[49] private __GAP;
+    uint256[46] private __GAP;
```

Comment:
Looks like GAP should be decreased by one additional?

Reply:
yup nice catch

Reply:
[fa04f06](https://github.com/Layr-Labs/eigenlayer-middleware/pull/311/commits/fa04f06212978dc3c66c23976cd34498f73aad41)